frontend/src/features/workspace/components/WorkspaceFormRow/RowActions/RowActionsDrawer.tsx

```diff
@@ -38,10 +38,10 @@ export const RowActionsDrawer = ({
     handleDeleteForm,
     handleDuplicateForm,
     handleEditForm,
-    handleManageFormAccess,
+    handleCollaborators,
```

Comment:
nit: should we rename this for clarity?
```suggestion
    handleViewCollaborators,
```